release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -74,6 +74,10 @@ Because {product-title} {product-version} now uses a {op-system-base} 9.2 based
 [id="ocp-4-14-installation-and-update"]
 === Installation and update
 
+[id="ocp-4-14-aws-shared-vpc"]
+==== Installing a cluster on Amazon Web Services (AWS) using a shared VPC
+In {product-title} {product-version}, you can install a cluster on AWS that uses a shared VPC, with a private hosted zone in a different account than the cluster. For more information, see xref:../installing/installing_aws/installing-aws-vpc.adoc#installing-aws-vpc[Installing a cluster on AWS into an existing VPC].
```

Comment:
Not critical, but {product-version} really should just be 4.14. The only time you should use {product-version} is if it would be correct to get bumped up to the next version in the next release. In the case of 4.14 release notes, it really isn't appropriate. But since we will use a different file/different content for 4.15, it really won't matter. 

And I know it's done a lot in the file already, but still recommend hardcoding to 4.14. :)

Reply:
Good point, I will hardcode it. Thank you